From 328dd2dba38c688a587d2857bc5cdd7af276b307 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 19 Jan 2021 10:27:52 +0100
Subject: [PATCH 251/529] drm/panel: rm68200: Add flags info

Return more info on bus & mode flags.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: Ibbcd2f0f8ad6cc23018aac6e3ced37bdd0abaea6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/191192
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228179
---
 drivers/gpu/drm/panel/panel-raydium-rm68200.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/gpu/drm/panel/panel-raydium-rm68200.c
+++ b/drivers/gpu/drm/panel/panel-raydium-rm68200.c
@@ -91,7 +91,7 @@ static const struct drm_display_mode def
 	.vsync_start = 1280 + 12,
 	.vsync_end = 1280 + 12 + 5,
 	.vtotal = 1280 + 12 + 5 + 12,
-	.flags = 0,
+	.flags = DRM_MODE_FLAG_NHSYNC | DRM_MODE_FLAG_NVSYNC,
 	.width_mm = 68,
 	.height_mm = 122,
 };
@@ -347,6 +347,8 @@ static int rm68200_get_modes(struct drm_
 
 	connector->display_info.width_mm = mode->width_mm;
 	connector->display_info.height_mm = mode->height_mm;
+	connector->display_info.bus_flags = DRM_BUS_FLAG_DE_HIGH |
+					    DRM_BUS_FLAG_PIXDATA_DRIVE_POSEDGE;
 
 	return 1;
 }
